compiler/damlc/daml-doc/src/DA/Daml/Doc/Render/Rst.hs

```diff
@@ -17,110 +18,170 @@ import           Data.Text.Prettyprint.Doc.Render.Text (renderStrict)
 import           Data.Char
 import           Data.Maybe
 import qualified Data.Text as T
+import qualified Data.Set as Set
 
 import CMarkGFM
 
-renderAnchor :: Maybe Anchor -> T.Text
-renderAnchor Nothing = ""
-renderAnchor (Just anchor) = "\n.. _" <> unAnchor anchor <> ":\n"
+-- | Renderer output. This is the set of anchors that were generated, and a
+-- list of output functions that depend on that set. The goal is to prevent
+-- the creation of spurious anchors links.
```

Comment:
Does _spurious_ refer to links that refer to locations that do not exist?

Reply:
Yes -- anchors that could exist but don't. I'll try to make this more clear.